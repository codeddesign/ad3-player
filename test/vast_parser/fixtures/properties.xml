<?xml version="1.0" encoding="UTF-8"?>
<VAST xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0">
</VAST>