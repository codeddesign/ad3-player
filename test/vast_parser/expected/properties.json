{
  "vast": {
    "name": "VAST",
    "value": "",
    "attributes": {
      "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
      "version": "2.0"
    }
  }
}